base/util.go

```diff
@@ -1058,7 +1058,7 @@ func Crc32cHash(input []byte) uint32 {
 }
 
 func Crc32cHashString(input []byte) string {
-	return fmt.Sprintf("0x%x", Crc32cHash(input))
+	return fmt.Sprintf("0x%08x", Crc32cHash(input))
```

Comment:
This function is also used by `GenerateIndexName` in `base/dcp_sharded.go` - changing the padding in this use-case will have implications for pre-existing CBGT indexes.
Given a new index name, import processing will not be able to resume from the last checkpoint, would reprocess documents from sequence zero, and leave old non-padded index around.

My preferred approach would be to have `GenerateIndexName` do its own non-padding formatting on `Crc32cHash` to maintain the existing behaviour, and include a godoc comment on this `Crc32cHashString` function to clarify that it now returns a padded 8 char hex string.